### PR TITLE
rems.application-list: truncate long descriptions

### LINKED ISSUE #1174: LBR: Applications table layout breaks when userid is too long
The View button is rendered outside the visible area.

Example:

application-table-wide

See also #1230

### PR DESCRIPTION
**Pull request by opqdonut**
fixes #1174

### CODE REVIEW

File: src/cljs/rems/application_list.cljs [hunk 1]
```diff
@@ -17,6 +17,14 @@
        (map localized)
        (str/join ", ")))
 
+(defn- format-description [app]
+  [:div {:style {:overflow :hidden
```
Comment (Macroz) [inline src/cljs/rems/application_list.cljs]: I'd rather put these in `styles.clj`.

Comment (opqdonut) [reply src/cljs/rems/application_list.cljs]: done. also simplified the styling a bit

**Comment by opqdonut** [comment]
we can factor out the overflow style later when we know what we need

**Comment by opqdonut** [comment]
this was for #1230

**Pull request merged**